package.json

```diff
@@ -42,9 +41,12 @@
         "karma-ie-launcher": "1.0.0",
         "karma-mocha": "1.3.0",
         "karma-mocha-reporter": "2.2.5",
+        "mini-css-extract-plugin": "^0.4.0",
```

Comment:
Please add a .yarnrc to prevent prefixes going into the package.json

Reply:
This was fixed by rebasing against master and removing the `^` by hand.

Master had the .yarnrc file with an empty string used as the prefix to add "on save".